backend-config/types.go

```diff
@@ -69,6 +69,10 @@ type SourceT struct {
 	WriteKey                   string
 	DgSourceTrackingPlanConfig DgSourceTrackingPlanConfigT
 	Transient                  bool
+	EventSchemasEnabled        bool
```

Comment:
EventSchemasEnabled?
```suggestion
```

Reply:
This was a merge conflict resolution issue. Fixed it. 👍🏼